_data/sites.json

```
@@ -8299,6 +8299,16 @@
         ]
     },
 
+    {
+        "name": "WT.Social",
+        "url": "https://wt.social/myaccount/deactivate-account",
+        "difficulty": "easy",
```

```suggestion
        "difficulty": "impossible",
```
If data cannot be fully deleted, we deem it impossible to delete the account

---

ok